### PR TITLE
refactor EBS service

### PR DESCRIPTION
**Pull request by xh3b4sd**
I am thinking to put the tccp and ebsvolume resources together. They need the master instance and ebs volume magic. The code could be a bit simplified IMO so I would go down that road when there is no veto. This PR here is only some minor refactoring.

### CODE REVIEW

File: service/controller/v25/ebs/ebs.go [hunk 1]
```diff
@@ -78,50 +81,56 @@ func (e *EBS) DeleteVolume(ctx context.Context, volumeID string) error {
...
+			},
 		}
 
-		e.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("stopped instance %s", attachment.InstanceID))
+		err := e.client.WaitUntilInstanceStopped(i)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		e.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("waited for instance %#q being stopped", attachment.InstanceID))
```
Comment (rossf7) [inline service/controller/v25/ebs/ebs.go]:
```suggestion
		e.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("waited for instance %#q to stop", attachment.InstanceID))
```

This shorter but still clear I reckon. WDYT?

**Comment by xh3b4sd** [comment]
So is the ebsvolume resource not only about the tccp? When it is also about the workers (tcdp), then merging is odd on a conceptual level.

File: service/controller/v25/ebs/ebs.go (hunk 1 above)
Comment (xh3b4sd) [reply service/controller/v25/ebs/ebs.go]: Yeah nice. I changed that. Makes it more consistent with the other logs above as well.

**Comment by rossf7** [comment]
I consulted the glossary <3. 

So I think both `ebsvolume` and `loadbalancer` resources relate to TCDP. They delete resources created by k8s that prevent us from deleting the worker nodes.

Maybe there is also some crossover with the control plane. IIRC the load balancer issue is it prevents us deleting the worker security group which prevents deleting the VPC.

**Comment by xh3b4sd** [comment]
Ok, I will have a think. I guess there is common logic to be executed across stack managing resources. We will see.

**Pull request merged**